From 2a5a279ac0192bc444cd1c3059169f576817d8b9 Mon Sep 17 00:00:00 2001
From: Moritz Warning <moritzwarning@web.de>
Date: Tue, 17 Sep 2024 11:50:37 +0200
Subject: [PATCH 5/5] remove noexecstack

The compilers for arm_cortex-a9 do not recognize this argument.
---
 make-linux.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/make-linux.mk
+++ b/make-linux.mk
@@ -364,7 +364,7 @@
 #override CXXFLAGS+=-fPIC -fPIE
 
 # Non-executable stack
-override LDFLAGS+=-Wl,-z,noexecstack
+override LDFLAGS+=-Wl,-z
 
 .PHONY: all
 all:	one
